MVP.md

```diff
@@ -11,3 +12,27 @@ In all of our investigations to-date, we have discovered one protocol which has
 
 The best solution we have found is a provider known as [Unlock Protocol](https://unlock-protocol.com/) which offers ERC-721 tokens that are fully configurable and can be implemented on a variety of common L1 and L2 blockchains.  From a pure functional perspective
 
+</details>
+
+# Problem = Signal 2 Noise Ratio
+
+<details><summary>The tokenomics of LexDAO are simple but the interactions are subtle:</summary>
+1. mmmber - annual fixed subscriptionin fiat but payable in stablecoins;
+   - working for membership - a loosely pegged conversion from base membership to LXCHAT, a timebank
+   - this timebank represents ~15 hrs of sweat as in-kind contribution in lieu of the annual membership subscription
+   - it is arguable whether these l'externs should have access to all guild channels given the semi-confidential nature of some projects but conceptually they have all access to the same platforms members have (discord, hackMD, github, etc)
+2. guest pass (discord only) - as a privilege members are permitted to issue guest passes for public to participate in special events within discord such as hackathons or study groups
+   - however most guest passes are automatically granted 4 wks via #introduction or by following the decision tree triage in #find-a-lawyer
+   - these guest passes are expoential backdown which means the 2nd renewal is at half the time
+   - they are automatically renewed if they submit a common which is "liked" by a LEETHodler or member performing the druid role (guest concierage)
+   - however, the same druids have power to revoke such guest privileges upon spam, behaviour contrary to community norms or upon complaint by regular member but which is appealable at @Ops total discretion
+3. LEETHodlers are a special case as there were certain historical discussions which grandfathered in certain privileges and retained as result of legacy (beware of cryptowizards for theur anger is slow but revenge is subtle.
+<!-- if people think this is unfair try growing a community of <40 through cryptowinter via unpaid vounteers sacrificing time as busy professionals, then come back to me -->
+</details>
+
+# solutiion = Fostering Interaction
+
+In addition to `CRED` which is the paidup membership token, we use sourceCred to
```

Comment:
It might be smarter to think of cred as reputation and membership as the baseline to be eligible for CRED.  I also believe that would be in line with what Jared intended.  At least in his larger mission for CRED.  We require a reputation system that is separate and distinct from the 721 membership card and which augments a user's reputation.  In this process is a substack of ontology and lexicon items that go alongside different permission gates for roles across the org.  As well as dashboards that can read and display these attributes clearly to other members.